portal/lib/src/ChatController.php

```diff
@@ -0,0 +1,138 @@
+<?php
+/**
+ *  Chat Class ChatController
+ *
+ * @package   OpenEMR
+ * @link      http://www.open-emr.org
+ * @author    Jerry Padgett <sjpadgett@gmail.com>
+ * @copyright Copyright (c) 2020 Jerry Padgett <sjpadgett@gmail.com>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+namespace PatientPortal;
```

Comment:
@sjpadgett ,
Do you want me to migrate these 3 classes into OpenEMR's main src directory. I would move them into src/PatientPortal/Chat/ directory and use namespace OpenEMR/PatientPortal/Chat and would then allow me to remove the 3 require_once commands that you are using to bring these in (the autoloader will take care of it).